### PR TITLE
send mail without tmp file

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

**Comment by ngilain** [comment]
:+1:
Beware of the rebase before merging

File: roles/orchestraCron/vars/main.yml [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
-robots_task: "{{command_php}} {{path_front}}/app/console {{robots_command}} 2>> {{error_file}}"
+robots_task: '{{command_php}} {{path_front}}/app/console {{robots_command}} {{mail_send_command}}'
 robots_command: "-e={{env}} orchestra:robots:generate"
 
-mail_minute: 59
-mail_hour: 0-23
-mail_task: "if [ -s '{{error_file}}' ]; then  cat {{error_file}} | {{sendmail_command}} fi; rm {{error_file}};"
-sendmail_command: mailx -s \"cron error\" {{notification_mail}};
-notification_mail: 'contact@open-orchestra.com'
+mail_to: cron@open-orchestra.com
```
Comment (nicolasThal) [inline roles/orchestraCron/vars/main.yml]: You should move this in the project configuration

File: roles/orchestraCron/vars/main.yml [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
+mail_from: 'contact@open-orchestra.com (OpenOrchestra)'
```
Comment (nicolasThal) [inline roles/orchestraCron/vars/main.yml]: likewise

**Comment by itkg-rboessel** [comment]
https://github.com/open-orchestra/open-orchestra/pull/522/files

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**